r/R/dplyr-mutate.R

```diff
@@ -112,7 +113,15 @@ mutate.Dataset <- mutate.ArrowTabular <- mutate.RecordBatchReader <- mutate.arro
 
 transmute.arrow_dplyr_query <- function(.data, ...) {
   dots <- check_transmute_args(...)
-  dplyr::mutate(.data, !!!dots, .keep = "none")
+  has_null <- vapply(dots, quo_is_null, logical(1))
```

Comment:
This is rather stylistic, but I think `map_lgl(dots, quo_is_null)` would fit a bit more with that package style (I see quite a few more `purrr::map`s compared to `vapply`s)

Reply:
good to know. i wondered about this